sdk/textanalytics/azure-ai-textanalytics/src/samples/java/com/azure/ai/textanalytics/lro/AnalyzeHealthcareEntitiesAsync.java

```diff
@@ -75,26 +75,41 @@ public static void main(String[] args) {
                         // Healthcare entities
                         healthcareEntitiesResult.getEntities().forEach(healthcareEntity -> {
                             System.out.printf(
-                                "\ti = %d, Text: %s, category: %s, subcategory: %s, confidence score: %f.%n",
-                                ct.getAndIncrement(), healthcareEntity.getText(), healthcareEntity.getCategory(),
+                                "\ti = %d, Text: %s, normalized name: %s, category: %s, subcategory: %s, confidence score: %f.%n",
+                                ct.getAndIncrement(), healthcareEntity.getText(),
+                                healthcareEntity.getNormalizedText(), healthcareEntity.getCategory(),
                                 healthcareEntity.getSubcategory(), healthcareEntity.getConfidenceScore());
+
+                            HealthcareEntityAssertion assertion = healthcareEntity.getAssertion();
+                            if (assertion != null) {
+                                System.out.printf(
+                                    "\tEntity assertion: association=%s, certainty=%s, conditionality=%s.%n",
```

Comment:
idk if this is ok in the sample, but the fact that `assertion` is not null, doesn't mean that all 3 properties will be populated. it could be one, or 2 or all

Reply:
Yes. Here is more on listing all of properties in the assertion even though some of them are null value.